test/option_test.rb

```diff
@@ -7,12 +7,12 @@ def Option(*args)
   end
 
   # proc
-  it { Option( ->(*args) { "proc! #{args.inspect}" } ).(1,2).must_equal "proc! [1, 2]" }
+  it { Option( ->(*args, **) { "proc! #{args.inspect}" } ).(1,2).must_equal "proc! [1, 2]" }
   it { Option( lambda { "proc!" } ).().must_equal "proc!" }
 
   # proc with instance_exec
-  it { Option( ->(*args) { "#{self.class} #{args.inspect}" } ).(Object, 1, 2).must_equal "OptionTest [Object, 1, 2]" }
-  it { Option( ->(*args) { "#{self} #{args.inspect}" }, instance_exec: true ).(Object, 1, 2).must_equal "Object [1, 2]" }
+  it { Option( ->(*args, **) { "#{self.class} #{args.inspect}" } ).(Object, 1, 2).must_equal "OptionTest [Object, 1, 2]" }
+  it { Option( ->(*args, **) { "#{self} #{args.inspect}" }, instance_exec: true ).(Object, 1, 2).must_equal "Object [1, 2]" }
```

Comment:
Do we need the `, **`?

Reply:
Yes. By not mentioning it, Option gives different results for `args.inspect` between ruby < 2.7 and  ruby 3.

```ruby
->(*args) { "#{args.inspect}" }.  # ruby < 2.6 => "Object [1, 2, {}]"
->(*args) { "#{args.inspect}" }.  # ruby >= 2.7 => "Object [1, 2]"
```